index.html

```diff
@@ -22,6 +24,29 @@
         </nav>
     </header>
 
+    <section class="primary-slider group">
+            <div class="primary-slider__content left">
```

Comment:
Since `.primary-slider__content` class has `position: absolute`, probably you don't need a float left?